protected/controllers/PatientController.php

```diff
@@ -1668,11 +1668,19 @@ private function performPatientSave(Contact $contact, Patient $patient, Address
                         }
                     }
                 } else {
+                    $address->validate();
                     // patient or address failed to save
                     $transaction->rollback();
 
                 }
             } else {
+                //to show validation error messages to the user
+                $address->validate();
+                $patient->validate();
```

Comment:
We might have to test this a bit more. $patient->validate() was what caused the original issue